app/controllers/catalog_controller.rb

```diff
@@ -1033,7 +1052,7 @@ def ot_copy_submit_add
         :description => @edit[:new][:description],
         :type        => old_ot.type,
         :content     => params[:template_content],
-        :draft       => @edit[:new][:draft] == "true" ? true : false)
+        :draft       => @edit[:new][:draft] == true || @edit[:new][:draft] == "true" ? true : false)
```

Comment:
A ruby expression `@edit[:new][:draft] == "true"` will return a boolean `true` or `false`.  There is no need to "convert" to true/false